bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -66,25 +71,15 @@ def predict(self):
         """
         raise NotImplementedError
 
-    def get_candidates(self):
-        """
-        Runs neural network and returns results for all classes. (Cached)
-        """
-        return [self.get_ball_candidates(), self.get_goalpost_candidates()]
-
-    def get_ball_candidates(self):
+    def get_candidates(self, class_name):
         """
         Runs neural network and returns results for ball class. (Cached)
```

Comment:
```suggestion
        Runs neural network and returns results for all classes. (Cached)
```
i assume you meant this?